From c6a5efa07e3088ed764a6fbdf1e6c05e0a4c8a56 Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Fri, 27 Jan 2023 17:27:19 -0800
Subject: [PATCH 11/18] tests: Fix a format error on darwin

test-compositor.c:94:25: error: format specifies type 'long' but the argument has type '__darwin_suseconds_t' (aka 'int') [-Werror,-Wformat]
                 getpid(), tv.tv_sec, tv.tv_usec);
                                      ^~~~~~~~~~

Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
---
 tests/test-compositor.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test-compositor.c b/tests/test-compositor.c
index 8ec0631..eb44a4f 100644
--- a/tests/test-compositor.c
+++ b/tests/test-compositor.c
@@ -93,7 +93,7 @@ get_socket_name(void)
 
 	gettimeofday(&tv, NULL);
 	snprintf(retval, sizeof retval, "wayland-test-%d-%ld%ld",
-		 getpid(), tv.tv_sec, tv.tv_usec);
+		 (int)getpid(), (long)tv.tv_sec, (long)tv.tv_usec);
 
 	return retval;
 }
-- 
2.24.3 (Apple Git-128)

